Detectors/AOD/src/aod-producer-workflow.cxx

```diff
@@ -36,6 +36,7 @@ void customize(std::vector<ConfigParamSpec>& workflowOptions)
     {"disable-root-output", o2::framework::VariantType::Bool, false, {"disable root-files output writer"}},
     {"disable-mc", o2::framework::VariantType::Bool, false, {"disable MC propagation"}},
     {"disable-secondary-vertices", o2::framework::VariantType::Bool, false, {"disable filling secondary vertices"}},
+    {"disable-trd-pid", o2::framework::VariantType::Bool, true, {"disable trd pid"}},
```

Comment:
The default of bool option should be false, please change this to `enable-trd-pid`